evalml/automl/automl_search.py

```diff
@@ -471,19 +492,10 @@ def __init__(
                     estimator,
                     self.problem_type,
                     parameters=self._frozen_pipeline_parameters,
-                    custom_hyperparameters=parameters,
                     sampler_name=self._sampler_name,
                 )
                 for estimator in allowed_estimators
             ]
-        else:
-            for pipeline in self.allowed_pipelines:
-                if self.pipeline_parameters:
-                    if pipeline.custom_hyperparameters:
-                        for component_name, params in self.pipeline_parameters.items():
-                            pipeline.custom_hyperparameters[component_name] = params
-                    else:
-                        pipeline.custom_hyperparameters = self.pipeline_parameters
```

Comment:
Good riddance! What I wanna say to this code

![get_outta_here](https://user-images.githubusercontent.com/235870/120863771-1c26d400-c559-11eb-9d18-089ce8af613b.gif)

😆